integration_tests/pyproject.toml

```diff
@@ -19,7 +19,7 @@ grpcio = "^1.33.2"
 PyYAML = "^5.3.1"
 python-dateutil = "^2.8.1"
 pystarport = {path = "../pystarport"}
-cryptopy = {git="https://github.com/crypto-com/cryptopy.git", rev = "fe444873294ed47de505596e307c4311c1489beb"}
+chainlibpy = {git="https://github.com/crypto-org-chain/chainlibpy.git", rev = "fe444873294ed47de505596e307c4311c1489beb"}
```

Comment:
can just use the version from PyPi now: https://pypi.org/project/chainlibpy/ ?

Reply:
changed, waiting test in CI pipeline to pass ⏳

Reply:
somehow the module name still need to be `cryptopy` when importing features, not that familiar with python yet, guess it's coined in the published module somewhere? might need @linfeng-crypto  to fix it in another PR.

FYI this commit https://github.com/crypto-org-chain/chain-main/pull/454/commits/93ed1429e67140829930f38e3e5a8fd917fb6f7a made integration tests pass

Reply:
It looks like it's because the integration tests use the lock file and the lock file is pinned against this commit sha1: fe444873294ed47de505596e307c4311c1489beb where it was called `cryptopy` ?

Reply:
https://github.com/crypto-org-chain/chain-main/pull/454#discussion_r608333064